install_config/aggregate_logging.adoc

```diff
@@ -1032,17 +1033,10 @@ the xref:../install/configuring_inventory_file.html#configuring-ansible[default
 For example:
 
 ----
-LOGGING_FILE_AGE=30
-LOGGING_FILE_SIZE=1024000
+$ oc set env ds/logging-fluentd LOGGING_FILE_PATH=console LOGGING_FILE_AGE=30 LOGGING_FILE_SIZE=1024000"
 ----
 
-After changing these parameters, re-run the xref:../install_config/aggregate_logging.adoc#deploying-the-efk-stack[logging installer playbook]:
-
-----
-$ cd /usr/share/ansible/openshift-ansible
-$ ansible-playbook [-i </path/to/inventory>] \
-    playbooks/openshift-logging/config.yml
-----
+To turn off log rotation, if the `LOGGING_FILE_PATH=console` is set and the `DEBUG` environment variable exists. With log rotation off, all logs will be printed to console but then they would be redirected to *_/var/log/fluentd.log file_*.
```

Comment:
@jcantrill please confirm

Reply:
Wouldn't it be easier to say:
"Turn off log rotation by setting `LOGGING_FILE_PATH=console`.  This will cause fluentd to write logs to STDOUT where they can be retrieved using the normal 'oc logs $fpod' command"

I don't believe its important for anyone to know the logs will go anyplace else but STDOUT.  The presumption here, IMO, is that when you write to console, use `oc logs`, when writing to file, use `oc exec $fpod logs`